lib/rules/require-capitalized-comments.js

```diff
@@ -52,7 +52,7 @@
  * ```
  *
  * ```js
- * "requireCapitalizedComments": { allExcept: ["jshint"] }
+ * "requireCapitalizedComments": { "allExcept": ["jshint"] }
```

Comment:
You are correct, but in order to update the website we need to pull these changes manually, we will release 1.13.1 today and i will update site with it